echemdb/make_pages.py

```diff
@@ -106,23 +100,17 @@ def create_element_pages(elementname):
     templatemd['data']['element'] = elementname
     templatemd['title'] = 'echemdb - {} CV data'.format(elementname)
 
-    target = copy.deepcopy(TARGET_FOLDERS['elements']).replace('tobesubstituted', elementname)
-    targetfile = TARGET_FOLDERS['path'] + target
-    os.makedirs(os.path.dirname(targetfile), exist_ok=True)
-    with open(targetfile, 'w') as f:
-        frontmatter.dump(templatemd, targetfile)
```

Comment:
No need to create the directories explicitly here. `mkdocs_gen_files.open` takes care of this for us, but only if we actually write to its `f` which we did not do before.